pkgs/development/tools/analysis/pev/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, openssl, fetchFromGitHub, ... }:
+stdenv.mkDerivation {
+  name = "pev-unstable-2018-07-22";
+  buildInputs = [ openssl.dev ];
+  phases = [ "unpackPhase" "buildPhase" "installPhase" ];
```

Comment:
Why do you skip the fixupPhase here? Does it still build if you remove this file?